gh-138191: Document ``frame.f_generator`` in the data model (#138540)
python/cpython: Doc/reference/datamodel.rst

     - Points to the previous stack frame (towards the caller),
       or ``None`` if this is the bottom stack frame

   * - .. attribute:: frame.f_generator
     - Returns the generator or coroutine object that owns this frame,

>        or ``None`` if the frame is of a regular function.

Reviewer: Why not at the end?

Author: @AA-Turner Is the placement correct now?

     - The "precise instruction" of the frame object
       (this is an index into the :term:`bytecode` string of the
       :ref:`code object <code-objects>`)
   
   * - .. attribute:: frame.f_generator

>      - Returns the generator or coroutine object that owns this frame,

Reviewer 2: Please add a cross reference to the generators page.

Author: See also :ref:`generators`.
@ZeroIntensity Could you please tell me what should be the label for the page.

>        or ``None`` if the frame is of a regular function.

Reviewer: ```suggestion
     - The :term:`generator` or :term:`coroutine` object that owns this frame,
       or ``None`` if the frame is a normal function.
```

>        or ``None`` if the frame is a normal function.

Reviewer: ```suggestion
       or ``None`` if the frame is a normal function.

       .. versionadded:: 3.14
```

     - The "precise instruction" of the frame object
       (this is an index into the :term:`bytecode` string of the
       :ref:`code object <code-objects>`)

>    * - .. attribute:: frame.f_generator

Reviewer: ```suggestion

   * - .. attribute:: frame.f_generator
```

     - The "precise instruction" of the frame object
       (this is an index into the :term:`bytecode` string of the
       :ref:`code object <code-objects>`)

>    

Reviewer: ```suggestion

```
